<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketOrdersWidget</class>
 <widget class="QWidget" name="MarketOrdersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sellOrdersTab">
      <attribute name="title">
       <string>Sell Orders</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="MarketOrdersTable" name="sellOrdersTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="buyOrdersTab">
      <attribute name="title">
       <string>Buy Orders</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="MarketOrdersTable" name="buyOrdersTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="TypePixmapLabel" name="typePixmapLabel">
       <property name="text">
        <string>Drag a type here.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="typeNameLabel">
           <property name="text">
            <string>&lt;h2&gt;No Item Selected&lt;/h2&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="InfoButton" name="infoButton" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MarketOrdersTable</class>
   <extends>QTableWidget</extends>
   <header>marketorderstable.hpp</header>
  </customwidget>
  <customwidget>
   <class>InfoButton</class>
   <extends>QWidget</extends>
   <header>infobutton.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TypePixmapLabel</class>
   <extends>QLabel</extends>
   <header>typepixmaplabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
